website/docs/reference/resource-configs/docs.md

```diff
@@ -45,6 +48,9 @@ This property is not implemented for sources.
 
 <TabItem value="seeds">
 
+You can use the docs property in YAML files, including the `dbt_project.yml`:
+
+
```

Comment:
What if we remove all the changes related to `website/docs/reference/resource-configs/docs.md` out of this PR and put them into a separate one instead?

We could provide code examples for `dbt_project.yml` similar to [`enabled`](https://docs.getdbt.com/reference/resource-configs/enabled)

Here's some code that should work:

```yaml
name: "my_project"
version: "1.0.0"
config-version: 2
profile: "sandcastle"


models:
  my_project:
    +docs:
      show: false

seeds:
  my_project:
    +docs:
      show: false

snapshots:
  my_project:
    +docs:
      show: false
```

So the generic code examples would look like this:

```yaml
models:
  [<resource-path>](/reference/resource-configs/resource-path):
    +docs:
      show: true | false

seeds:
  [<resource-path>](/reference/resource-configs/resource-path):
    +docs:
      show: true | false

snapshots:
  [<resource-path>](/reference/resource-configs/resource-path):
    +docs:
      show: true | false
```

Reply:
hey @dbeatty10 , i moved it out of this pr and added it here: https://github.com/dbt-labs/docs.getdbt.com/pull/4529